CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Mobly Release History
 
+# Mobly Release 1.13: SL4A Removal and Test Suite Improvements
+Removed all SL4A related code. Improved test suite mechanism.
+
+### New
+* Support test case selection and listing for test suites.
+* Support selecting test cases within single test class using regular
+  expressions.
+* Support `fastboot` command execution with updated serial and customized binary
```

Comment:
"updated" is ambiguous.

As we discussed, the key is that a remove/virtual device's identifier may change during a test.

Reply:
Thans, rephrased. PLAT